### DISCUSSION TITLE
TV Replay

### DISCUSSION THREAD

**Discussion by cheznewa** [iptv-org/community#187] [Ideas]
Hi/Hello, My Idea Is Add Replay Playlist, Of The World, In Case You Missed Watch.
Like :::: Captvty.

**Comment by Dum4G** [comment]
Sadly not possible for 99.99% of this repo because it should be supported serverside. And our playlist comes from thousands of different sources that are only intended to show just current livestream.

**Reply by cheznewa** [reply]
Write scripts for generate playlist?

**Comment by BellezaEmporium** [comment]
Our repository is dedicated to live streams, not video on demand.
